src/core_plugins/console/api_server/es_5_0/indices.js

```diff
@@ -217,4 +217,18 @@ module.exports = function (api) {
       "{indices}/_upgrade"
     ]
   });
+
+  api.addEndpointDescription('_close', {
+    methods: ["POST"],
+    patterns: [
+      "{indices}/_close"
+    ]
+  });
+
+  api.addEndpointDescription('_open', {
+    methods: ["POST"],
+    patterns: [
+      "{indices}/_open"
```

Comment:
Normally we expect single quotes for most strings, but I see this file is already using quotes inconsistently, so we'll pick it up in a future bulk pass.